### ISSUE TITLE
Flashed 1.9 to Seeed Wio, stuck at startup screen

### ISSUE THREAD

**Issue by circuitmike** [disk91/WioLoRaWANFieldTester#53]
I got the Seeed Wio bundle a while back and thought it was time for a firmware update. I flashed 1.9 (LoRaE5) to the Wio and now it simply hangs at the startup screen ("Wio LoRaWan Field Tester / Version v1.9 (LoRaE5) / WIO_FT_XXXXXXXXXX"). On the LoRa section there's a steady blue light and blinking red light. Any idea what might be going on?

I've got enough experience with embedded systems and the like to suspect the power issue, so I connected it to a power supply with plenty of current to spare and it still does the same thing.

**Comment by disk91** [comment]
Hello, 
That looks strange ...
I propose you flash it again ...

If you stil have the same issue please DM me on discord we will investigate this.

**Comment by disk91** [comment]
any update ?

**Comment by circuitmike** [comment]
Flashing again was the first thing I tried, and it did exactly the same thing. I'll try to find you on Discord one of these days when I've got some time.

**Comment by circuitmike** [comment]
I'm leaving this closed because I finally figured out I had a hardware problem. The 5-way switch on the front of my Wio Terminal has failed and at least one of the 5 switches is constantly bouncing. This causes the device to go into and immediately out of configuration mode at startup, causing a boot loop. Once in a while if I hold the button just right I can get it to start up, but that's it. I got a new Wio Terminal but unfortunately I don't see a way to recover the app key provided by Seeed.
